video2dataset/workers/subset_worker.py

```diff
@@ -160,7 +158,24 @@ def process_shard(
                 meta["clips"] = [[line_dict["start"], line_dict["end"]] for line_dict in subtitles]
 
             elif self.detect_cuts:  # apply cut detection to get clips
-                meta["cuts"] = self.cut_detector(streams)
+                video_bytes = streams["video"]
+                downsampled_video_bytes, error_message = self.cut_detector_downsampler([video_bytes])
+
+                if error_message is not None:
```

Comment:
I think you can remove this, not obvious to me this is how we want to do it, we should start thinking about how to do error handling better

Reply:
unless there's a particular reason you put it here